java/src/main/resources/com/ncc/aif/ontologies/InterchangeOntology

```diff
@@ -579,13 +579,21 @@
     rdfs:range xsd:string ;
     rdfs:subPropertyOf owl:topDataProperty .
 
+#:claimSemantics rdf:type owl:ObjectProperty ;
+#    rdfs:label "claim semantics" ;
+#    rdfs:comment "Event/Relation KEs that describe inner claim" ;
+#    rdfs:domain :Claim ;
+#    rdfs:range [ owl:unionOf (
+#        :Event
+#        :Relation ) ] ;
+#    rdfs:subPropertyOf owl:topObjectProperty .
+
 :claimSemantics rdf:type owl:ObjectProperty ;
     rdfs:label "claim semantics" ;
     rdfs:comment "Event/Relation KEs that describe inner claim" ;
```

Comment:
This comment doesn't look like it applies any longer since Event and Relation have been changes to SameAsCluster.

Reply:
Yeah, I left this one here because I was worried this may still be up for debate.  This was a NIST request, but wasn't sure how the teams would take this.

Reply:
Updated